src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -270,24 +270,19 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
           <div key={`${room.title}-${index}`}>{renderSelectedRoom(index)}</div>
         ) : (
           <div
-            className="Container__room-preview"
+            className={classNames("Container__room-preview", {
+              "Container__room-image--disabled": !room.isEnabled,
+            })}
             style={{
               top: `${room.y_percent}%`,
               left: `${room.x_percent}%`,
               width: `${room.width_percent}%`,
               height: `${room.height_percent}%`,
+              backgroundImage: "url(" + room.image_url + ")",
```

Comment:
.

Reply:
fixed in 3874ad43e2155b00a686bba8122dd8f5041a5fb3